hadoop-ozone/dist/src/main/compose/ozonesecure/docker-config

```diff
@@ -80,12 +80,11 @@ HDFS-SITE.XML_rpc.metrics.percentiles.intervals=60,300
 OZONE_DATANODE_SECURE_USER=root
 SECURITY_ENABLED=true
 KEYTAB_DIR=/etc/security/keytabs
-KERBEROS_KEYTABS=dn om scm HTTP testuser testuser2 s3g
 KERBEROS_KEYSTORES=hadoop
 KERBEROS_SERVER=kdc
 JAVA_HOME=/usr/lib/jvm/jre
 JSVC_HOME=/usr/bin
 SLEEP_SECONDS=5
 KERBEROS_ENABLED=true
 
-no_proxy=om,scm,s3g,kdc,localhost,127.0.0.1
\ No newline at end of file
+no_proxy=om,scm,s3g,kdc,localhost,127.0.0.1
```

Comment:
should we have recon in the no_proxy list?

Reply:
AFAIK it was added to support the the curl/wget executed by the smoketests. As we have no recon specific acceptance tests (yet, hopefully), it's not immediately required...

Reply:
There is one recon acceptance tests just added recently to catch a regression in HDDS-2897.
See #453 for details.

Reply:
This was added in another task, only newline at EOF is changed.  So I think it should be addressed in a new Jira.